docs/examples/basic_conceptual_guide/Part_1-model_deployment/Readme.md

```diff
@@ -0,0 +1,293 @@
+```
+cd /navigate/to/this/folder
+```
+
+#### Text Detection
+Download and unzip OpenCV's EAST model.
+```
+wget https://www.dropbox.com/s/r2ingd0l3zt8hxs/frozen_east_text_detection.tar.gz
+tar -xvf frozen_east_text_detection.tar.gz
+```
+
+Export to ONNX. The [NGC TensorFlow 2 container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow) can also be used as an environment.
+```
+pip install tensorflow
+pip install -U tf2onnx
+python -m tf2onnx.convert --input frozen_east_text_detection.pb --inputs "input_images:0" --outputs "feature_fusion/Conv_7/Sigmoid:0","feature_fusion/concat_3:0" --output model.onnx
+```
+
+#### Text Recognition
+Download the Text Recognition model weights. Use the [NGC PyTorch container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pytorch) as the environment for the following.
+
+```
+docker run -it --gpus all -v ${PWD}:/scratch nvcr.io/nvidia/pytorch:<yy.mm>-py3
+cd /scratch
+wget https://www.dropbox.com/sh/j3xmli4di1zuv3s/AABzCC1KGbIRe2wRwa3diWKwa/None-ResNet-None-CTC.pth
+```
+
+Export the models as `.onnx` using the file in the `utils` folder. This file is adapted from [Baek et. al. 2019](https://github.com/clovaai/deep-text-recognition-benchmark).
+```
+import torch
+from utils.model import STRModel
+
+# Create PyTorch Model Object
+model = STRModel(input_channels=1, output_channels=512, num_classes=37)
+
+# Load model weights from external file
+state = torch.load("None-ResNet-None-CTC.pth")
+state = {key.replace("module.", ""): value for key, value in state.items()}
+model.load_state_dict(state)
+
+# Create ONNX file by tracing model
+trace_input = torch.randn(1, 1, 32, 100)
+torch.onnx.export(model, trace_input, "str.onnx", verbose=True)
+```
+
+### Setting up the model repository
+
+A model repository is a filesystem based repository of models and configuration schema used by the Triton Inference Server. While a server can use one or more model repositories, for the purposes of simplicity, this explanation only uses a single repository. The general structure of a model repository is as follows:
+```
+<model-repository>/
+  <model-name>/
+    [config.pbtxt]
+    [<output-labels-file> ...]
+    <version>/
+      <model-definition-file>
+    <version>/
+      <model-definition-file>
+    ...
+  <model-name>/
+    [config.pbtxt]
+    [<output-labels-file> ...]
+    <version>/
+      <model-definition-file>
+    <version>/
+      <model-definition-file>
+    ...
+  ...
+```
+
+There are three important components to be discussed from the above structure:
+* `model name`: The identifying name for the model.
+* `config.pbtxt`: For each model, users need to define the model configuration. This configuration, at minimum, needs to define the name of the platform, and the name, shape, and datatype of the input and output layers.
+* `version`: versioning allows for multiple versions of the same model to be available for use depending on the policy selected.
```

Comment:
Nit: Consistent capitalization for this list ("Versioning"). Also, breaking up the sentence a bit more, e.g. "available for use, depending on the version policy." (And link to it here: https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_configuration.md#version-policy)

Configurations are now optional for some models, by the way. If Triton can read the model via its framework fully and you don't want to make any customizations from the default autocomplete. I'd also break up the second sentence a bit more for readability, e.g. This configuration, at minimum, needs to define: the platform and the name, shape, and datatype of model inputs and outputs.

Reply:
I think I added the autocomplete bit somewhere in the tutorial, but missed it here, will change it.